mobileversion.html

```diff
@@ -1,236 +1,265 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-     <meta charset="UTF-8" >
-     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-     <link rel="stylesheet" href="mobileversion.css">
-     <link rel="preconnect" href="https://fonts.googleapis.com">
-     <link rel="preconnect" href="https://fonts.googleapis.com">
-     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-     <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
-     <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.2/css/all.css">
-     <title>Mobile Version Website</title>
-  </head>
-    <body>
-       <header class="header">
-        <nav>
-     <a href="#">My Logo</a>
-     <a href="#"><i class="fa fa-bars"></i></a>
-     </nav>
-        </header>
-             <section>
-                <div class="flex-container">
-                    <div class="item5">
+
+<head>
+   <meta charset="UTF-8">
+   <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+   <link rel="stylesheet" href="mobileversion.css">
+   <link rel="preconnect" href="https://fonts.googleapis.com">
+   <link rel="preconnect" href="https://fonts.googleapis.com">
+   <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+   <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+   <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.2/css/all.css">
+   <title>Mobile Version Website</title>
+</head>
+
+<body>
+   <section class="mobileversion">
+      <header class="header">
+         <nav class="nav-bar">
+            <a class="log" href="#">My Logo</a>
+            <ul class="nav-menu">
+
+               <li class="nav-item"><a href="#Portfolio" class="nav-link">Portfolio</a></li>
+               <li class="nav-item"><a href="#About" class="nav-link">About</a></li>
+               <li class="nav-item"><a href="#Contact" class="nav-link">Contact</a></li>
+               <hr>
+            </ul>
+            <div class="hamburger">
+               <span class="bar"></span>
+               <span class="bar"></span>
+               <span class="bar"></span>
+            </div>
+         </nav>
+      </header>
+      <section>
```

Comment:
- [ OPTIONAL] I think it would be great if you follow the typography( font, color) and spacing(marign, padding) in order to stick to the figma template and follow the design. 
#### Your design👇
![menust](https://user-images.githubusercontent.com/66992901/201896308-b2b44646-113a-4901-b581-2e55d5f96169.PNG)
#### figma template👇
![menufigma](https://user-images.githubusercontent.com/66992901/201896373-1f911218-3f9a-44d7-9f05-751ac0fa8469.png)